### PR TITLE
added feature to provide display name different from product name for…

### PR DESCRIPTION
**Pull request by santanu33**
… API Product

### CODE REVIEW

**Comment by santanu33** [comment]
I signed it!

**Comment by SantanuSahoo2017** [comment]
I signed it!

**Comment by santanu33** [comment]
I signed it!

File: lib/commands/createproduct.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ function createProduct(opts,request,done){
 	}
 
 	product.name = opts.productName
-	product.displayName = opts.productName
+	product.displayName = opts.displayName
```
Comment (theganyo) [inline lib/commands/createproduct.js]: It looks like this value will always be null as the [descriptor](https://github.com/apigee/apigeetool-node/blob/d165bbc6d66194a0ad7efba85d718ecd7f6ebc2a/lib/commands/createproduct.js#L10) that declares the CLI options doesn't include `displayName` as one of the options. Has this been tested?

Comment (SantanuSahoo2017) [reply lib/commands/createproduct.js]: added `displayName` options in CLI options.

**Comment by whitlockjc** [comment]
LGTM _(so long as it's tested)_

@theganyo Can you take another peek?

**Comment by theganyo** [comment]
LGTM

**Comment by whitlockjc** [comment]
This has been released in `apigeetool@0.9.2`.

Thanks for your contribution!

**Pull request merged**